app/views/sidebar/_dashboard.html.erb

```diff
@@ -18,13 +18,20 @@
   </div>
   
 <% if current_user %>
-  <div style="margin: 2.2rem 0;">
-    <a class="btn btn-lg btn-outline-secondary blurred-location-input" submit_to="/profile/tags/create/<%= current_user.uid %>"><i class="fa fa-map-marker" style="color:#c40;"></i> Add your location</a>
-    <p style="font-size: smaller; margin: 0.3rem 0;">
-      Add your location to see work near you<br>
-      <a href="https://publiclab.org/location-privacy" target="_blank">About location privacy »</a>
-    </p>
-  </div>
+  <% if current_user.has_power_tag("lat") && current_user.has_power_tag("lon") && current_user.has_power_tag("zoom") %>
+    <div style="margin: 2.2rem 0;">
+      <a class="btn btn-lg btn-outline-secondary" href="https://publiclab.org/map/#<%= current_user.get_value_of_power_tag("zoom").to_i%>/<%=current_user.get_value_of_power_tag("lat").to_f %>/<%=current_user.get_value_of_power_tag("lon").to_f %>">
+      <i class="fa fa-map-marker" style="color:#c40;"></i> Nearby Activity</a>
+    </div>
+  <%else %>
+    <div style="margin: 2.2rem 0;">
+      <a class="btn btn-lg btn-outline-secondary blurred-location-input" submit_to="/profile/tags/create/<%= current_user.uid %>"><i class="fa fa-map-marker" style="color:#c40;"></i> Add your location</a>
+      <p style="font-size: smaller; margin: 0.3rem 0;">
+        Add your location to see work near you<br>
+        <a href="https://publiclab.org/location-privacy" target="_blank">About location privacy »</a>
```

Comment:
`target="_blank"` will open this in a new window or tab, right? 
@jywarren this is expected behaviour?

Reply:
It should, but it doesn't in my local. When I click on `Nearby Activity` it doesn't open up a new tab, rather it continues in the same tab. I don't know what the expected behaviour should be :disappointed_relieved: , maybe @jywarren can tell us more?